www/page.php

```diff
@@ -4,8 +4,8 @@
 <title><?=_('Level0 OpenStreetMap Editor') ?></title>
 <meta charset="utf-8">
 <meta name="generator" content="<?=GENERATOR ?>">
-<link rel="stylesheet" href="https://unpkg.com/leaflet@0.7.7/dist/leaflet.css" />
-<script src="https://unpkg.com/leaflet@0.7.7/dist/leaflet.js"></script>
+<link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css" integrity="sha256-p4NxAoJBhIIN+hmNHrzRCf9tD/miZyoHS5obTRR9BMY=" crossorigin="" />
```

Comment:
Do those `integrity` and `crossorigin` things actually work? My vim editor highlights those as warnings.

Reply:
It works. Change the integrity hash and the script will refuse to load. https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity